cloud/terraform/terraform_controller.py

```diff
@@ -23,18 +26,25 @@ def get_instances(self):
         output = os.popen('terraform show --json').read()
         json_output = json.loads(output)
 
-        resources = json_output['values']['root_module']['resources_tf']
+        resources = json_output['values']['root_module']['resources']
 
         instances_info = {}
         if self.cloud == 'aws':
             # 'address' corresponds to the tf resource id
             for resource in resources:
+                if 'aws_instance' not in resource['address']:
+                    continue
+
+                username = self.tf_configurator.get_username_by_instance_name(
+                    resource['address'].replace('aws_instance.', '')
```

Comment:
I would do a split using the dot, and then get the second item of the array.
```suggestion
                    resource['address'].split('.')[1]
```